app/src/main/java/org/openobservatory/ooniprobe/domain/GenerateAutoRunServiceSuite.java

```diff
@@ -38,7 +39,7 @@ public AbstractSuite generate(
             OONISession session = EngineProvider.get().newSession(
                     EngineProvider.get().getDefaultSessionConfig(
                             app,
-                            BuildConfig.SOFTWARE_NAME,
+                            BuildConfig.SOFTWARE_NAME + "-unattended",
```

Comment:
I see you factored a constant for `AUTORUN`. Would it make sense to factor one also for `"-unattended"`?